File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/dart/practicas_3/registro.dart
import 'dart:io';

void main () {

  print('Numero de personas para el registro:');
  int cuantos = int.parse(stdin.readLineSync()!);

  int cont1 = 0;
  int cont2 = 0;

  for (int vueltas = 1; vueltas <= cuantos; vueltas++) {

  print('----------------------------------------------------------');

  print('Bienvenido al registro de la empresa, por favor ponga sus datos acontinuacion:');

  print('----------------------------------------------------------');

  print('Nombre y apellido:');
  String name = stdin.readLineSync()!;

  print('----------------------------------------------------------');

  print('Numero de documento:');
  int document = int.parse(stdin.readLineSync()!);

  print('----------------------------------------------------------');

  print('Codigo de verificacion de la empresa:');
  int pass = int.parse(stdin.readLineSync()!);

  print('----------------------------------------------------------');

  

  if (pass == 777){
    print('El usuario $name, Numero de documento $document a sido registrado correctamente.');
    cont1 = cont1 + 1;
  } else {
    print('El usuario no puede registrarse por falta de codigo de verificacion.');
    cont2 = cont2 + 1;
  }

  print('----------------------------------------------------------');

  }

  print('INFORMACION TOTAL DE LOS REGISTRADOS:');

  print('----------------------------------------------------------');

  print('Numero de personas registradas:');
  print(cont1);

  print('----------------------------------------------------------');

  print('Numero de personas que no pasaron el registro:');
  print(cont2);

  print('----------------------------------------------------------');

}

File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/ruby/practicas_3/registro.rb
puts "Numero de personas para el registro:"
cuantos = gets.chomp.to_i

cont1 = 0
cont2 = 0

for vueltas in 1..cuantos

    puts "----------------------------------------------------------"

    puts "Bienvenido al registro de la empresa, por favor ponga sus datos acontinuacion:"

    puts "----------------------------------------------------------"

    puts "Nombre y apellido:"
    name = gets.chomp

    puts "----------------------------------------------------------"

    puts "Numero de documento:"
    document = gets.chomp.to_i

    puts "----------------------------------------------------------"

    puts "Codigo de verificacion de la empresa:"
    pass = gets.chomp.to_i

    puts "----------------------------------------------------------"


    if pass == 777
        puts "El usuario #{name}, Numero de documento #{document} a sido registrado correctamente."
        cont1 = cont1 + 1
    else
        puts "El usuario no puede registrarse por falta de codigo de verificacion."
        cont2 = cont2 + 1
    end

end

puts "----------------------------------------------------------"

puts "INFORMACION TOTAL DE LOS REGISTRADOS:"

puts "----------------------------------------------------------"
puts "Numero de personas registradas:"
puts cont1

puts "----------------------------------------------------------"
puts "Numero de personas que no pasaron el registro:"
puts cont2

puts "----------------------------------------------------------"